types/seccomp.go

```diff
@@ -2,12 +2,20 @@ package types
 
 // Seccomp represents the config for a seccomp profile for syscall restriction.
 type Seccomp struct {
-	DefaultAction Action     `json:"defaultAction"`
-	Architectures []Arch     `json:"architectures"`
-	Syscalls      []*Syscall `json:"syscalls"`
+	DefaultAction Action         `json:"defaultAction"`
+	Architectures []Arch         `json:"architectures,omitempty"`
+	ArchMap       []Architecture `json:"archMap,omitempty"`
```

Comment:
This looks like a bit of duplication (or I don't really get it :sweat_smile:), but I guess this is for retro-compatibility right ? (if yes, a comment about that would be perfect :stuck_out_tongue_closed_eyes: )